tools/zulint/command.py

```diff
@@ -98,6 +98,9 @@ def run_linter():
                     # invoking the external linter.
                     return 0
 
+            if fix:
+                command.append('--fix')
```

Comment:
This can access the new `self.args.fix` to check whehter `--fix` was used at the top level.